CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 -->
+## Unreleased
+### Changes
+- ([\#720](https://github.com/desmos-labs/desmos/pull/720)) Integrated CosmWasm 1.0.0-beta module
+
```

Comment:
No need to do this, we rely only on `changeset` entries. The CHANGELOG will be generated from those